docs/guides/multi-regional-setup-dedicated-regions/README.md

```diff
@@ -1,85 +1,224 @@
+```
+Verify that `vmselect` pod is up and running by executing the following command:
+```sh
+kubectl get pods | grep global-vmselect
+```
 
-An additional VictoriaMetrics single can be set up in every region, scraping metrics from the main TSDB.
+The expected output is:
+```
+global-vmselect-victoria-metrics-cluster-vmselect-5f594665g66jb   1/1     Running   0          3m38s
+```
+## 5. Install vmauth in global route layer
+We also deploy `vmauth` service in global route layer. It can work as a gateway, distributing requests to multiple AZ-clusters. Key capabilities include load balancing and failover. See [vmauth](https://docs.victoriametrics.com/victoriametrics/vmauth/).
+
+If each AZ-cluster contains the full copy of data, then we can get the complete results from any AZ-cluster. The `global vmauth` can be configured with `first_available` mode to route user queries to the first available AZ-cluster.
+
+Prepare `vmauth.yaml`:
+```yaml
+config:
+    unauthorized_user:
+      load_balancing_policy: first_available
+      url_map: 
+        - src_paths:
+            - "/select/.*"
+          url_prefix:
+            - "http://<az1-vmauth-addr>:8427"
+            - "http://<az2-vmauth-addr>:8427"
+            - "http://<az3-vmauth-addr>:8427"
+            - "http://<az4-vmauth-addr>:8427"
+podAnnotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "8427"
+# For more configurations, please refer to https://docs.victoriametrics.com/helm/victoria-metrics-auth/
+```
+Then run
+```
+helm install vmauth vm/victoria-metrics-auth -f vmauth.yaml
+```
+Verify that `vmauth` pod is up and running by executing the following command:
+```
+kubectl get pods | grep vmauth
+```
+The expected output is:
+```
+vmauth-victoria-metrics-auth-5b4b5bc869-prb5b                     1/1     Running   0          81s
+```
+`vmauth` runs on `:8427` by default.
 
-You also may evaluate the option to send these metrics to the neighbour region to achieve HA.
+## 6. Check the obtained result 
+We can query the metrics through `vmauth` in the global routing layer.
 
-Additional context
-* VictoriaMetrics Single - [https://docs.victoriametrics.com/victoriametrics/single-server-victoriametrics/#monitoring](https://docs.victoriametrics.com/victoriametrics/single-server-victoriametrics/#monitoring)
-* VictoriaMetrics Cluster - [https://docs.victoriametrics.com/victoriametrics/cluster-victoriametrics/#monitoring](https://docs.victoriametrics.com/victoriametrics/cluster-victoriametrics/#monitoring)
+```sh
+export POD_NAME=$(kubectl get pods --namespace default -l "app.kubernetes.io/instance=vmauth" -o jsonpath="{.items[0].metadata.name}")
+kubectl --namespace default port-forward $POD_NAME 8427
+```
 
+Execute the following command to get metrics via curl:
+```
+curl -sg 'http://127.0.0.1:8427/select/0/prometheus/api/v1/query_range?query=count(up{kubernetes_pod_name=~".*vmauth.*"})&start=-1m&step=10s' | jq
```

Comment:
1. Why do we want to demonstrate it?
2. It is impossible to use 127.0.0.1 since we are talking about Kubernetes. Let's use a placeholder.